File: programmers/level1/026.py
#26. 문자열 다루기 기본
def solution(s):
    if len(s) == 4 or len(s) == 6:
        answer = True
    else: answer = False
    
    if answer:
        for i in s:
            if i.isalpha():
                answer=False
                break
    return answer

File: programmers/level0/026.java
// 26. 배열 자르기
import java.util.*;
class Solution {
    public int[] solution(int[] numbers, int num1, int num2) {
        return Arrays.copyOfRange(numbers, num1, num2+1);
        /* int[] answer = new int[num2-num1+1];
        for(int i=num1; i<=num2; i++){
            answer[i-num1]=numbers[i];
        }
        return answer;*/
    }
}